### PR TITLE
Fix resolve issue flakes in CI e2e tests.

### PR DESCRIPTION
**Pull request by miheer**
Fix resolve issue flakes in CI e2e tests by using `net.LookupHost` method in the `verifyExternalIngressController` method.

=== NAME  TestAll/parallel/TestAWSEIPAllocationsForNLB
    util_test.go:641: waiting for loadbalancer domain apps.eiptest.ci-op-d2nddmn0-43abb.origin-ci-int-aws.dev.rhcloud.com to resolve...
    util_test.go:641: waiting for loadbalancer domain apps.eiptest.ci-op-d2nddmn0-43abb.origin-ci-int-aws.dev.rhcloud.com to resolve...
    lb_eip_test.go:119: loadbalancer domain apps.eiptest.ci-op-d2nddmn0-43abb.origin-ci-int-aws.dev.rhcloud.com was unable to resolve:

### CODE REVIEW

File: test/e2e/util_test.go [hunk 1]
```diff
@@ -632,25 +632,26 @@ func verifyExternalIngressController(t *testing.T, name types.NamespacedName, ho
...
-				t.Logf("waiting for loadbalancer domain %s to resolve...", address)
+				t.Logf("%v waiting for HTTP route name %s to resolve (%v)", time.Now(), address, err)
 				return false, nil
 			}
 			return true, nil
 		}); err != nil {
-			t.Fatalf("loadbalancer domain %s was unable to resolve:", address)
+			t.Fatalf("HTTP route name %s was unable to be resolved: %v", address, err)
 		}
 	}
```
Comment (Miciah) [inline test/e2e/util_test.go:648]: `assertHttpRouteConnection` has similar logic.  Can you factor this logic out into a separate `waitForHostToResolve` helper, and use that helper in both `assertHttpRouteConnection` and `verifyExternalIngressController`?

**Review by Miciah — COMMENTED** [review]
Please follow conventions for the commit message.

**Comment by gcs278** [comment]
@miheer I have also opened https://github.com/openshift/cluster-ingress-operator/pull/1127 as an alternate way resolve these E2E flake issues.

Arguably, https://github.com/openshift/cluster-ingress-operator/pull/1127 is more consistent with our other usages of `verifyExternalIngressController`, and doesn't require us to crank up the timeout to really large times.

**Pull request closed without merging**